frontend/src/OneBuilding/AcademyButtons.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  View, Text, Image, StyleSheet, TouchableHighlight,
+} from 'react-native';
+import { Entypo } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+
+import { SubCardView } from '../common/components';
+import goldIcon from '../../assets/gold.png';
+import troopIcon from '../../assets/troop/troop.png';
+
+const styles = StyleSheet.create({
+  container: {
+    paddingVertical: 12,
+  },
+  row: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  textStyle: {
+    fontSize: 14,
+    color: '#fff',
+  },
+  imageStyle: {
+    width: 32,
+    height: 32,
+  },
+  iconStyle: {
+    width: 16,
+    height: 16,
+  },
+  description: {
+    marginLeft: 12,
+  },
+});
+
+function AcademyButtons({ createTroops, upgrade }) {
+  return (
+    <View style={styles.container}>
+      <TouchableHighlight onPress={createTroops} style={{ padding: 3 }}>
+        <SubCardView>
+          <Image resizeMode="contain" source={troopIcon} style={styles.imageStyle} />
+          <View style={styles.description}>
+            <Text style={styles.textStyle}>Create troop level 1</Text>
```

Comment:
the level of the troops should come from the Academy level. a Level 1 Academy creates level 1 troops, a level 2 Academy creates level 2 troops.... at least it's logical for me. Please discuss in the team, if you want every level to create level 1 troops that is also fine for me, but then what would be the point in upgrading the Academy if it doesn't create stronger troops?